duckencoder.py

```diff
@@ -238,44 +250,26 @@ def keyInstr2USBBytes(keyinstr, keyProp, langProp):
-                        elif keyinstr == "CONTROL":
-                                keyinstr == "CTRL"
-                        elif keyinstr == "DOWNARROW":
-                                keyinstr == "DOWN"
-                        elif keyinstr == "UPARROW":
-                                keyinstr == "UP"
-                        elif keyinstr == "LEFTARROW":
-                                keyinstr == "LEFT"
-                        elif keyinstr == "RIGHTARROW":
-                                keyinstr == "RIGHT"
-                        elif keyinstr == "MENU":
-                                keyinstr == "APP"
-                        elif keyinstr == "WINDOWS":
-                                keyinstr == "GUI"
-                        elif keyinstr == "PLAY" or keyinstr == "PAUSE":
-                                keyinstr == "MEDIA_PLAY_PAUSE"
-                        elif keyinstr == "STOP":
-                                keyinstr == "MEDIA_STOP"
-                        elif keyinstr == "MUTE":
-                                keyinstr == "MEDIA_MUTE"
-                        elif keyinstr == "VOLUMEUP":
-                                keyinstr == "MEDIA_VOLUME_INC"
-                        elif keyinstr == "VOLUMEDOWN":
-                                keyinstr == "MEDIA_VOLUME_DEC"
-                        elif keyinstr == "SCROLLLOCK":
-                                keyinstr == "SCROLL_LOCK"
-                        elif keyinstr == "NUMLOCK":
-                                keyinstr == "NUM_LOCK"
-                        elif keyinstr == "CAPSLOCK":
-                                keyinstr == "CAPS_LOCK"
-                        else:
-                                keyinstr = keyinstr[0:1].upper()
+                        keyinstr = keyinstr.strip().upper()
+                        keyinstr = {"ESCAPE": "ESC",
+                                    "DEL": "DELETE",
+                                    "BREAK": "PAUSE",
+                                    "CONTROL": "CTRL",
+                                    "DOWNARROW": "DOWN",
+                                    "UPARROW": "UP",
+                                    "LEFTARROW": "LEFT",
+                                    "RIGHTARROW": "RIGHT",
+                                    "MENU": "APP",
+                                    "WINDOWS": "GUI",
+                                    "PLAY": "MEDIA_PLAY_PAUSE",
+                                    "PAUSE": "MEDIA_PLAY_PAUSE",
+                                    "STOP": "MEDIA_STOP",
+                                    "MUTE": "MEDIA_MUTE",
+                                    "VOLUMEUP": "MEDIA_VOLUME_INC",
+                                    "VOLUMEDOWN": "MEDIA_VOLUME_DEC",
+                                    "SCROLLLOCK": "SCROLL_LOCK",
+                                    "NUMLOCK": "NUM_LOCK",
+                                    "CAPSLOCK": "CAPS_LOCK"}.get(keyinstr, keyinstr[0:1])
```

Comment:
Looks very clean. `get(keyinstr, keyinstr[0:1]) ` should be `get(keyinstr, keyinstr[0:1].upper())`.
This assures `GUI r` in script is translated to `KEY_R` instead of `KEY_r`